features/achordion.c

```diff
@@ -274,7 +291,7 @@ __attribute__((weak)) bool achordion_eager_mod(uint8_t mod) {
 }
 
 #ifdef ACHORDION_STREAK
-__attribute__((weak)) uint16_t achordion_streak_timeout(uint16_t tap_hold_keycode) {
+__attribute__((weak)) uint16_t achordion_streak_timeout(uint16_t tap_hold_keycode, uint16_t next_keycode) {
```

Comment:
For backwards compatibility, I suggest 

```
__attribute__((weak)) uint16_t achordion_streak_chord_timeout(uint16_t tap_hold_keycode, uint16_t next_keycode) {
  return achordion_streak_timeout(tap_hold_keycode);
}

/** @deprecated Use `achordion_streak_chord_timeout()` instead. */
__attribute__((weak)) uint16_t achordion_streak_timeout(uint16_t tap_hold_keycode) {
  return 100;
}
```

This way, new users can use `achordion_streak_chord_timeout()` while existing users can continue to build using `achordion_streak_timeout()`. I never know how many users there really are (such is open source), so going for backwards compatibility where reasonably possible seems worthwhile.

Reply:
Ahh, good idea.

> return achordion_streak_timeout(tap_hold_keycode);

This technically wouldn't be backwards compatible, since at the moment that function gets passed the key _before_ the tap-hold key (I think I messed this up in the PR description and wrote 'after'). I'd consider it a bugfix though, their config might suddenly start doing what they intended to begin with. There isn't a way at the moment to emulate the old behavior of that function, that's the last feature I've been considering adding where you can exempt certain keys from starting the streak timer.

Reply:
Sorry, I was imprecise: I meant compatibility in the sense of keeping existing keymaps building. Those are good points on difference in behavior and to consider that as a bug fix.

Reply:
I put this in. Old configs should still compile now with slightly different behavior.